backEnd/src/themes/themes.controller.ts

```diff
@@ -49,4 +49,32 @@ export class ThemesController {
   create(@Body() body: CreateThemeDto) {
     return this.themesService.create(body);
   }
+
+  @ApiOperation({ summary: 'Update a theme' })
+  @ApiBody({
+    type: CreateThemeDto,
+    description: 'Update a theme',
+    required: true,
+  })
+  @ApiResponse({
+    status: 201,
+    description: 'The theme has been successfully updated.',
+  })
+  @ApiResponse({
+    status: 422,
+    description: 'The theme not found.',
+  })
+  @Put()
+  @HttpCode(201)
+  update(@Body() body: CreateThemeDto) {
+    return this.themesService.update(body);
+  }
```

Comment:
For put actions, we also receive the entity id via params

```suggestion
  @Put(':id')
  @HttpCode(201)
  update(@Param('id') id: string, @Body() body: UpdateThemeDto) {
    return this.themesService.update(id, body);
  }
```